### ISSUE TITLE
Update styling on old components to no longer need `tjenester3.css`

### ISSUE THREAD

**Issue by bjosttveit** [Altinn/app-frontend-react#858]
### Description

This file originally came from the old deprecated design system. It was copied over in app-frontend to make WCAG related modifications. There are several components that use classes defined here, and they should rather get their own css-modules so that `src/styles/tjenester3.css` can be deleted.

**Comment by bjosttveit** [comment]
This was a weird problem that only seemed to affect that specific app.
https://altinn.slack.com/archives/C02EJ9HKQA3/p1685703534127879

There was a css-rule in tjenester3.css that would give `.form-group` (which is used by `GenericComponent`) `display: hidden` on `@media print`. We should really get rid of this old global styling to prevent such weird things happening in the future.
